README.md

```diff
@@ -329,8 +329,8 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | Lean                          | [asdf-community/asdf-lean](https://github.com/asdf-community/asdf-lean)                                         | [![Build Status](https://github.com/asdf-community/asdf-lean/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/asdf-community/asdf-lean/actions)                                                             |
 | Leiningen                     | [miorimmax/asdf-lein](https://github.com/miorimmax/asdf-lein)                                                   | [![Build Status](https://travis-ci.org/miorimmax/asdf-lein.svg?branch=master)](https://travis-ci.org/miorimmax/asdf-lein)                                                                                                         |
 | Levant                        | [asdf-community/asdf-hashicorp](https://github.com/asdf-community/asdf-hashicorp)                               | [![Build](https://github.com/asdf-community/asdf-hashicorp/actions/workflows/workflow.yml/badge.svg)](https://github.com/asdf-community/asdf-hashicorp/actions/workflows/workflow.yml)                                            |
-| LFE                           | [vic/asdf-lfe](https://github.com/vic/asdf-lfe)                                                                 | [![Build Status](https://travis-ci.org/vic/asdf-lfe.svg?branch=master)](https://travis-ci.org/vic/asdf-lfe)                                                                                                                       |
-| Link (system tools)           | [vic/asdf-link](https://github.com/vic/asdf-link)                                                               | [![Build Status](https://travis-ci.org/vic/asdf-link.svg?branch=master)](https://travis-ci.org/vic/asdf-link)                                                                                                                     |
```

Comment:
This one doesn't appear to be in the `plugins/` dir. Can you confirm?

Reply:
I don't see it there, but I have no idea what this is and/or whether it should be. This is the first time I'm using `asdf` and just figured I'd contribute these small redirect fixes.

Or are you asking someone else to confirm it?

Reply:
The readme is not what is used by `asdf`, but rather just information for users to look at to ctrl+f search for tools etc. `asdf` uses the files in `plugins/*` dir to download the plugins. The "Link" plugin you've updated here doesn't have a matching file in `plugins/*` so won't actually be installable via this repo

Reply:
That makes sense. What would you like me to do in this PR then?

I could:
* not update that entry in the README
* remove it from the README
* add it a file in the plugins directory (not sure what process you follow, but that's probably best in a separate PR?)
* something else?

Reply:
I mean, if it exists, add it.